tests/configs/config_ds000248.py

```diff
@@ -23,6 +23,10 @@
 noise_cov = 'emptyroom'
 
 spatial_filter = 'ssp'
+n_proj_eog = dict(n_mag=1, n_grad=1, n_eeg=1)
+n_proj_ecg = dict(n_mag=1, n_grad=1, n_eeg=0)
+ecg_proj_from = 'evoked'
+eog_proj_from = 'epochs'
```

Comment:
I find this very unintuitive @hoechenberger @agramfort -- if we average or not does not depend on on whether it's epochs or evoked does it? I would find a simple average parameter more intuitive, matching the MNE API.

Reply:
@dengemann you would prefer ecg_proj_epochs_average= bool ?

Personally I prefer typing True or False but no strong feeling. A book is also closer to MNE API.

Reply:
yes I would. For the same reasons, hence, my proposal.